comp/vision/gamepiece_finder.py

```diff
@@ -13,18 +13,15 @@
 class GamePieceFinder:
```

Comment:
maybe you could make all these changes in a copy of the file, call it notefinder.py or something.

the reason is that for ease of deployment we put *everything* in one file, so it's good for you to have your own file to work in.